vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -15,6 +15,17 @@
 					      	<div style="height: 5px; width: 5px; border-radius:50%; background-color: var(--success)"></div>
 					      	<em style="color: var(--dark-grey); font-size: 12px; margin-left: 10px">logged in as <span th:text="${#authentication.name}"></span></em>
 					    </div>
+		        	</li>
+		        	<a  th:href="@{/exhibit/about}">
+		        	<span th:if="${aboutPageTitle!=null && !#strings.isEmpty(aboutPageTitle)}" class = "icon-info secondary" />     	
+	        		<span class="custom_Text"
+                                   th:if="${aboutPageTitle!=null && !#strings.isEmpty(aboutPageTitle)}"
+                                   th:text="About"
+                                   style="font-size: 18px; cursor: pointer; font-weight: bold;
+                                   margin-right:10px; margin-left:-10px; padding-right:10px; margin-top:10px"
+                                   >
```

Comment:
the about text should be on its own page

Reply:
This is the button to redirect to the About page which is its own page